mapper/transaction.go

```diff
@@ -63,6 +72,86 @@ func Transaction(
 	traceOps := traceOps(flattenedTrace, len(feeOps))
 	ops = append(ops, traceOps...)
 
+	for _, transferLog := range transferLogs {
+		// ERC721 index the value in the transfer event.  ERC20's do not
+		if len(transferLog.Topics) == topicsInErc721Transfer {
+			// Only parse Erc721 when setting is enabled
+			if !parseErc721 {
+				continue
+			}
+
+			contractInfo, err := client.ContractInfo(transferLog.Address, false)
+			if err != nil {
+				return nil, err
+			}
+
+			// Don't include default tokens if setting is not enabled
+			if !includeUnknownTokens && contractInfo.Symbol == clientTypes.UnknownERC721Symbol {
+				continue
+			}
+
+			contractAddress := transferLog.Address
+			addressFrom := transferLog.Topics[1]
+			addressTo := transferLog.Topics[2]
+			erc721Index := transferLog.Topics[3] // Erc721 4th topic is the index.  Data is empty
+			receiptOp := types.Operation{
+				OperationIdentifier: &types.OperationIdentifier{
+					Index: int64(len(ops)),
+				},
+				Status:  types.String(StatusSuccess),
+				Type:    OpErc721Transfer,
+				Amount:  Erc721Amount(erc721Index, contractAddress, contractInfo.Symbol, contractInfo.Decimals, false),
+				Account: Account(ConvertHashToAddress(&addressTo)),
+			}
+			sendingOp := types.Operation{
```

Comment:
Nit: you are going to want to populate the `related_operations` field for both 20 and 721

Reply:
This comment still stands?